app/views/shared/_footer_lite.html.slim

```diff
@@ -3,9 +3,9 @@ footer.footer.bg-navy.white
     .clearfix
       .sm-col = t('shared.footer_lite.gsa')
       .sm-col-right.caps
-        = link_to t('links.help'), help_path,
+        = link_to t('links.help'), "https://#{APP_NAME}/help",
```

Comment:
Oh gotcha - sure! that clarifies. Or, we could have some type of constant like `MARKETING_SITE_URL` to clarify what's going on here since in my head the IDP was always `login.gov`